modules/ROOT/pages/first-steps-after-setup.adoc

```diff
@@ -106,7 +106,7 @@ NOTE: do not create a user or an administrator with the same login and password
 . Click on the "More" button (in the top right corner).
 . Under "Users mapping", click on "Add a user".
 . Select your user and click on the "Add" button. Log out as the Technical user and log back in as the newly created user with administrative rights.
```

Comment:
click on the "Add" button. => click on "Add" button.

Reply:
Sure, I'm tired to fight this removal of "the" from everywhere. I'll remove it